README.md

```diff
@@ -64,10 +70,17 @@ $ export FLASK_APP=superset
 $ export SUPERSET_CONFIG_PATH=/path/to/superset_config.py
 ```
 
-Initialize the database. Create an administrator. Create default roles
+Set this environment variable to allow OAuth 2.0 authentication with
+CommCare HQ over insecure HTTP. (DO NOT USE THIS IN PRODUCTION.)
+```bash
+$ export AUTHLIB_INSECURE_TRANSPORT=1
```

Comment:
Is this and `OAUTHLIB_INSECURE_TRANSPORT` needed if we are using OAuth with CommCareHQ environment available locally?
If we are providing this option to communicate with live CommCareHQ environments, we probably should not.